string sentence
int32 count
---
string result